src/common/playlists/PlayHeaderActions.jsx

```diff
@@ -41,6 +41,16 @@ const PlayHeaderActions = ({ play }) => {
           <AiOutlineRead className="icon" size="24px" />
           <span className="sr-only">Blog</span>
         </a>
+      ) : (
```

Comment:
I dont think we need a new icon. I suggest you just change the links conditionally based on the blog data is available or not.

Reply:
Ok